cmd/plugin.bash_completion.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# bash tab-completion
+
+# This is a per-library function definition, used in conjunction with the
+# top-level entry point in ign-tools.
+
+function _gz_plugin
+{
+  if [[ ${COMP_WORDS[COMP_CWORD]} == -* ]]; then
+    # Specify options (-*) word list for this subcommand
+    COMPREPLY=($(compgen -W "
+      -h --help
+      --help-all
+      --version
+      -v --verbose
+      -i --info
+      -p --plugin
```

Comment:
It will be tough to keep this in sync with the file that implements the flags. We should at least add a comment to both files that we should update one when updating the other. The best thing though would be to write an automated test that checks that the flags are the same on both.

Reply:
d5db3f9 + 1b21972

Reply:
Note that while ign-plugin2 is using CLI11, this PR targets Citadel, where ign-plugin1 is still the Ruby way. 

I tested on ign-plugin2 by copying the diffs from this PR, together with the ign-tools PR, and bash completion works. This new test passes in both plugin1 and plugin2.